bluepysnap/node_sets.py

```diff
@@ -111,12 +40,15 @@ def __init__(self, filepath):
             NodeSets: A NodeSets object.
         """
         self.content = utils.load_json(filepath)
-        self.resolved = _resolve(self.content)
+        self._instance = libsonata.NodeSets.from_file(filepath)
 
-    def __getitem__(self, node_set_name):
+    def resolve_ids(self, node_set_name, population):
```

Comment:
I am not sure if `resolve_ids` is an appropriate name, I would expect to call `resolve` to resolve internally and then `get` to get the actual value; so I think in this case calling it `get_ids` straight away might be a good choice

Reply:
yeah, valid point. `get_ids` or `ids` is better